back/Gemfile.lock

```diff
@@ -189,14 +189,6 @@ PATH
       rails (~> 6.1)
       verification
 
-PATH
```

Comment:
@SaraBCL Not sure where this change came from, I think it was automatically generated when starting the backend - should I revert or keep?

Reply:
I yes I had the same at some point ! To prevent from happening I think you need to pull master both in citizenlab and ee and re-build, but indeed it doesn't make sense to merge and should be reverted